qtconsole/frontend_widget.py

```diff
@@ -68,6 +68,9 @@ def highlightBlock(self, string):
         # the string as plain text so we can compare it.
         current_block = self.currentBlock()
         string = current_block.text()
+        # QTextBlock::text() can still return non-breaking spaces
```

Comment:
```suggestion

        # QTextBlock::text() can still return non-breaking spaces
```

Blank line for clarity.